front/app/modules/commercial/content_builder/admin/components/ContentBuilderSettings/index.tsx

```diff
@@ -17,6 +17,9 @@ import { getComponentNameMessage } from '../RenderNode';
 import messages from '../../messages';
 import { FormattedMessage } from 'utils/cl-intl';
 
+// events
+import eventEmitter from 'utils/eventEmitter';
```

Comment:
I assume you already discussed this with your squad, but why the use of of `eventEmitter`? I am not a huge fan of it, because

1. The components are not that far away from each other and it's not that hard to just pass around callback/handler functions
2. It makes the component less reusable. With a callback/handler it's explicit how to reuse it in a different context but here you have to know about the event emitter stuff when you are using the component, and what happens if you have multiple listeners?
3. It's more boilerplate-y, see e.g. the `content_builder/admin/components/ContentBuilderTopBar/index.tsx` component
4. We're only using this for highly exceptional situations in the rest of the codebase, like with the sign up modal. So it's a deviation from our standard way of doing things